github/repos_deployments.go

```diff
@@ -176,6 +175,29 @@ func (s *RepositoriesService) ListDeploymentStatuses(owner, repo string, deploym
 	return *statuses, resp, err
 }
 
+// GetDeploymentStatus returns a single deployment status of a repository.
+//
+// GitHub API docs: https://developer.github.com/v3/repos/deployments/#get-a-single-deployment-status
+func (s *RepositoriesService) GetDeploymentStatus(owner, repo string, deploymentID int, deploymentStatusID int) (*DeploymentStatus, *Response, error) {
```

Comment:
nit: I would combine the two `int` params just like the two `string` params are combined:

```go
func (s *RepositoriesService) GetDeploymentStatus(owner, repo string, deploymentID, deploymentStatusID int) (*DeploymentStatus, *Response, error) {
```